packages/trader/src/AppV2/Components/PurchaseButton/purchase-button.tsx

```diff
@@ -95,25 +96,23 @@ const PurchaseButton = observer(() => {
     }, [is_purchase_enabled]);
 
     if (is_accumulator && has_open_accu_contract) {
-        const info = proposal_info?.[trade_types_array[0]] || {};
+        const is_accu_sell_disabled = !is_valid_to_sell || active_accu_contract?.is_sell_requested;
         return (
             <div className='purchase-button__wrapper'>
                 <Button
                     color='black'
-                    variant='secondary'
                     size='lg'
-                    label={<Localize i18n_default_text='Sell' />}
+                    label={
+                        is_accu_sell_disabled
+                            ? `${cardLabels.CLOSE}`
+                            : `${cardLabels.CLOSE} ${current_stake} ${currency}`
+                    }
                     fullWidth
                     className='purchase-button purchase-button--single'
-                    disabled={!is_valid_to_sell || active_accu_contract?.is_sell_requested}
+                    disabled={is_accu_sell_disabled}
                     onClick={() => onClickSell(active_accu_contract?.contract_info.contract_id)}
-                >
-                    <PurchaseButtonContent
-                        {...purchase_button_content_props}
-                        info={info}
-                        current_stake={current_stake}
-                    />
-                </Button>
+                />
+                {is_accu_sell_disabled && <div className='purchase-button purchase-button--disabled-background' />}
```

Comment:
Note: An approach with `::before` pseudo-element for this white background didn't work because `::before` applied to the button becomes its child and its stacking context changes when there are other children present, which leads to `::before` appearing on top of the button instead of beneath it despite applying `z-index`.